src/common/plist.mligo

```diff
@@ -0,0 +1,98 @@
+// SPDX-FileCopyrightText: 2021 TQ Tezos
+// SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+#if !COMMON_PLIST_H
```

Comment:
I just realized that (based on how we did it until now) this module would be perhaps better located under the `src/proposal/` directory

Reply:
Updated this.